src/Sdk/Validation/Rules/Regex.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\SdkBlueprint\Sdk\Validation\Rules;
+
```

Comment:
`preg_match` may also return false on error, which is associated with a PHP Warning, however it would be a good idea to use !== 1 instead because the applications using this package may not always use the same error reporting levels.

Reply:
good point, fixed.